yb-voyager/cmd/exportDataStatusCommand.go

```diff
@@ -69,7 +69,9 @@ var exportDataStatusCmd = &cobra.Command{
 		}
 		useDebezium = msr.IsSnapshotExportedViaDebezium()
 
-
+		if msr.SourceDBConf == nil {
```

Comment:
There is a small window after sourceDBConf is created but before export data actually starts exporting tables. If export-data-status is called in that window what happens? 
Just confirming that we also have a check later to print the same message if status files are not yet created, right?

Reply:
Yes yes, that is already there